pulsar-broker/src/main/java/org/apache/pulsar/broker/admin/v2/Worker.java

```diff
@@ -115,6 +116,21 @@ public List<ConnectorDefinition> getConnectorsList() throws IOException {
         return workers().getListOfConnectors(clientAppId());
     }
 
+    @GET
+    @ApiOperation(
+            value = "Fetches the list of built-in Pulsar functions",
+            response = FunctionDefinition.class,
+            responseContainer = "List"
+    )
+    @ApiResponses(value = {
+            @ApiResponse(code = 200, message = "Get builtin functions successfully.")
+    })
+    @Path("/builtinfunctions")
```

Comment:
IMO if we want to follow the same as the "/connectors" above this endpoint should be "/functions"

Reply:
I think that "buildinfunctions" is clearer.
"functions" may look like to refer to "all the functions"

not a big deal